src/util/manager/interaction.ts

```diff
@@ -74,9 +77,10 @@ export default class InteractionManager extends Manager {
     try {
       await handler.handleInteraction({ bot: this.bot, interaction });
     } catch (e: any) {
-      await interaction.reply(
-        "Command execution failed. Please contact a bot maintainer..."
-      );
+      await interaction.reply({
+        content: "Command execution failed. Please contact a bot maintainer...",
+        ephemeral: true
```

Comment:
Made this ephemeral to hide failed executions from public, in case someones trying to use an anon slash command or smth.